### DISCUSSION TITLE
Sign of life

### DISCUSSION THREAD

**Discussion by McDutchie** [ksh93/ksh#698] [General]
I just wanted to show a sign of life after a three-month period of inactivity on my part. I have been dealing with an extended crisis in the family sphere. Things are normalising now and I expect to be able to resume my activity here early in the next year. In the mean time I wish everyone happy holidays and all the best for 2024.

**Comment by jghub** [comment]
first of all, I sincerely do hope that the mentioned family-related issues can be resolved in a good way (if that is possible, that is). 

otherwise: really happy to hear that you have not abandoned ksh! 

merry christmas and a good new year.

**Comment by phidebian** [comment]
Welcome back :-) and happy new year to all ksh fixers :-)
